### ISSUE TITLE
Allow uppercase words in the dictionaries

### ISSUE THREAD

**Issue by pablocostass** [vkatsuba/rebar3_sheldon#28]
## `rebar3_sheldon` version
[0.3.1](https://github.com/vkatsuba/rebar3_sheldon/releases/tag/0.3.1)

## `OS` version
macOS 11.6.1

## Description
* **Motivation**
When reading the dictionary associated with a project, one might sometimes find themselves for a second wondering what a word that is usually wrote in uppercase is, as you will find it in lowercase in the dictionary. For example, `asts` (`ASTs`).
* **Proposal**
Allow uppercase words in the dictionaries and lowercase ([`string:lowercase/1`](https://www.erlang.org/doc/man/string.html#lowercase-1)) them when reading the dictionary files.

**Comment by vkatsuba** [comment]
Hi @pablocostass, thanks for report but this is issue is related to [Sheldon](https://github.com/inaka/sheldon). Will take a look, thanks.

**Comment by pablocostass** [comment]
Oh man, I mixed up the repos! I can close this issue and re-open it there if you want to :)

**Comment by vkatsuba** [comment]
No-no, all is ok, I just put some mention about it here because - perhaps `Sheldon` will have two modes of operation, where the registry will be taken into account and where the registry will not be taken into account for such cases as you indicated.

**Comment by vkatsuba** [comment]
So, let's keep ticket here for now :upside_down_face:.
